src/Tools/TransformXMLToJSONUsingXSLT/App.config

```diff
@@ -0,0 +1,41 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <configSections>
+    <section name="Filters" type="System.Configuration.AppSettingsSection" />
+  </configSections>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7" />
+  </startup>
+  <appSettings>
+    <add key="xmlFilePathResult" value="TestResult-Color_Tests2.xml" />
+    <add key="xsltFile" value="TransformXMLToText.xslt" />
+    <add key="textFileComplete" value="textFileComplete.txt" />
+    <add key="textFileWithFilters" value="TextResultWithFilters.txt" />
+    <add key="applyFilters" value="1" /><!--1 means yes, 0 means not-->
+  </appSettings>
+
+  <Filters>
+    <add key="SerializationTests" value="SerializationTests" />
+    <add key="JSONSerializationTests" value="JSONSerializationTests" />
+    <add key="SerializationNonGuidIdsTest" value="SerializationNonGuidIdsTest" />
+    <add key="SerializationTest" value="SerializationTest" />
+    <add key="SerializationInDifferentCultureTest" value="SerializationInDifferentCultureTest" />
+    <add key="RecordedTestsDSEngine" value="RecordedTestsDSEngine" />
+    <add key="CoreTests" value="CoreTests" />
+    <add key="RecordedTests" value="RecordedTests" />
+    <add key="MigrationTestFramework" value="MigrationTestFramework" />
+    <add key="Regressions" value="Regressions" />
+    <add key="NodeViewCustomizationTests" value="NodeViewCustomizationTests" />
+    <add key="NodeMigrationTests" value="NodeMigrationTests" />
+    <add key="ComplexTests" value="ComplexTests" />
+    <add key="DSEvaluationModelTest" value="DSEvaluationModelTest" />
+  </Filters>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-8.0.0.0" newVersion="8.0.0.0" />
```

Comment:
a little odd - I wonder if this occurred when you added the new json 8.0.3 - can you make sure this tool builds and runs fine when you start with a clean bin and packages folder?

Reply:
This reference to Newtonsoft.Json won´t be needed.